### DISCUSSION TITLE
uuid without dash

### DISCUSSION THREAD

**Discussion by RobbiNespu** [Kong/insomnia#4896] [Help]
how I can manipulate tag uuid or custom tag `{% uuid 'v4' %}` to display uuid without dash?

**Comment by monecchi** [comment]
It'd be a great deal! I have to make a request with a unique id but the API endpoint forbides any special character! However, I've came across the "insomnia-plugin-custom-generated-value" that does the job quite well generating random values allowing a slightly control over it. It does not generate uuid though. You can check it here: https://insomnia.rest/plugins/insomnia-plugin-custom-generated-value

**Comment by ihexxa** [comment]
One option might be manipulating it in the pre-request script which will be released soon.
